SAPHana/bin/SAPHanaSR-manageAttr

```diff
@@ -670,31 +680,110 @@ chk_multi_target_requirements() {
     return $chk
 }
 
+# get resource name from cib
+get_res() {
+    local resType=$1
+    cib=/var/lib/pacemaker/cib/cib.xml
+    # get resources for given $RSID
+    resS="$(grep '<nvpair name="SID"' $cib | sed -n "/value=\"${RSID}\"/s/.*id=\"\(.*\)-instance_attributes-.*>/\1/gp")"
+    if [ -n "$INO" ]; then
+        # get resources for given $INO
+        resI="$(grep '<nvpair name="InstanceNumber"' $cib | sed -n "/value=\"${INO}\"/s/.*id=\"\(.*\)-instance_attributes-.*>/\1/gp")"
+    fi
+    # get cln or msl resource name for given $RSID
+    # if possible do not use 'crm configure show', even if it's nicer to parse
+    # it's much slower than direkt reading cib.xml
+    for rs in $resS; do
+        if [ -n "$INO" ]; then
+            for ri in $resI; do
+                if [ "$rs" == "$ri" ]; then
+                    pat="[A-Za-z/ ][[:space:]]Set:.*${rs}"
+                    rname=$(crm_mon -r1 2>/dev/null | awk -v pat="$pat" '$0 ~ pat { print $3 }')
+                    rtype=$(awk -v pat="id=\"${rname}\"" '$0 ~ pat { print substr($1,2,length($1)-1) }' $cib)
+                    # clone oder ms?
+                    if [ "$rtype" == "$resType" ]; then
+                        echo "$rname"
+                    fi
+                fi
+            done
+        else
+            pat="[A-Za-z/ ][[:space:]]Set:.*${rs}"
+            rname=$(crm_mon -r1 2>/dev/null | awk -v pat="$pat" '$0 ~ pat { print $3 }')
+            rtype=$(awk -v pat="id=\"${rname}\"" '$0 ~ pat { print substr($1,2,length($1)-1) }' $cib)
+            # clone oder ms?
+            if [ "$rtype" == "$resType" ]; then
+                echo "$rname"
+            fi
+        fi
+    done
+}
+
+# check, if the cluster or the msl and cln resource is in maintenance
+chk_for_maintenance() {
+    local mmaint="true"
+    local cmaint="true"
+    # check maintenance mode of cluster
+    # msl resource is reported unmanaged, but cln resource not
+    if crm_mon -r1 | grep "Resource management is DISABLED" >/dev/null 2>&1; then
```

Comment:
Autsch. And I tried to build a 4 node cluster with node names "Resource", "management", "is" and "DISABLED" ;-)
Just kidding of course, But we should make sure, the string "Resource management is DISABLED" can be used an API.